azurerm/resource_arm_automation_account.go

```diff
@@ -221,9 +262,21 @@ func flattenAutomationAccountSku(sku *automation.Sku) []interface{} {
 }
 
 func expandAutomationAccountSku(d *schema.ResourceData) *automation.Sku {
```

Comment:
and this

Reply:
This is for backward compatibility when both `sku` and `sku_name` exist side by side...